make/autoconf/basics_windows.m4

```diff
@@ -396,22 +430,7 @@ AC_DEFUN_ONCE([BASIC_COMPILE_FIXPATH],
       fixpath_argument_list=`echo $all_unique_prefixes  | tr ' ' '@'`
       FIXPATH="$FIXPATH_BIN -m$fixpath_argument_list"
     fi
-    FIXPATH_SRC_W="$FIXPATH_SRC"
-    FIXPATH_BIN_W="$FIXPATH_BIN"
-    BASIC_WINDOWS_REWRITE_AS_WINDOWS_MIXED_PATH([FIXPATH_SRC_W])
-    BASIC_WINDOWS_REWRITE_AS_WINDOWS_MIXED_PATH([FIXPATH_BIN_W])
-    $RM -rf $FIXPATH_BIN $FIXPATH_DIR
-    $MKDIR -p $FIXPATH_DIR $CONFIGURESUPPORT_OUTPUTDIR/bin
-    cd $FIXPATH_DIR
-    $CC $FIXPATH_SRC_W -Fe$FIXPATH_BIN_W > $FIXPATH_DIR/fixpath1.log 2>&1
-    cd $CURDIR
 
-    if test ! -x $FIXPATH_BIN; then
-      AC_MSG_RESULT([no])
-      cat $FIXPATH_DIR/fixpath1.log
-      AC_MSG_ERROR([Could not create $FIXPATH_BIN])
-    fi
-    AC_MSG_RESULT([yes])
```

Comment:
> So how are you supposed to get the fixpath.exe on windows/aarch64?
> 
> On the surface, this backport seem to have the same problem as your original win/aarch64 patch for mainline, that it is not generally possible to build it without a lot of external hacks. I can only point this out; I'll have to leave it to the jdk11u maintainers to accept or drop the patch given this knowledge.

Auto-testing is always going to be a pain as long as the port isn't bootstrappable because there aren't native tools, right?

Reply:
> So how are you supposed to get the fixpath.exe on windows/aarch64?

If you do not want to download an arbitrary `.exe` file as described mailing list post linked above (which I totally get), you can just do a build for Windows/x86 and copy over the `fixpath.exe` that was generated by that.

I understand having hacks around the build system issues is less than ideal. Ideally we would backport your WINENV patch to 11, but that would be much more invasive in terms of risk managment. The proposed solution in this PR does at least barely touch other platforms, so the risk for breaking something is rather low imho.